packages/client/components/TimelineHeader.tsx

```diff
@@ -104,6 +107,15 @@ const TimelineHeader = (props: Props) => {
         {timelineEventTypeFilterMenuPortal(
           <TimelineEventTypeMenu menuProps={timelineEventTypeFilterMenuProps} />
         )}
+        <button
+          className='mt-4 mr-16 mb-4 ml-0 flex items-center sidebar-left:mt-0 sidebar-left:mr-6 sidebar-left:mb-0 sidebar-left:ml-0'
+          onClick={() =>
+            history.push(constructFilterQueryParamURL(teamIds, null, !showArchived, eventTypes))
+          }
+        >
+          <div className='text-sm font-semibold text-slate-600'>{'Archived'}</div>
+          <Checkbox active={showArchived} className='ml-2' />
+        </button>
```

Comment:
-1 Something seems not right here on Safari
<img width="735" alt="image" src="https://github.com/user-attachments/assets/d4c9075d-7914-4eba-833d-ee428ec9a060">